src/andromede/simulation/output_values.py

```diff
@@ -247,3 +247,107 @@ def _are_mappings_close(
             )
     else:
         return True
+
+
+@dataclass(frozen=True)
+class BendersSolution:
+    data: Dict[str, Any]
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, BendersSolution):
+            return NotImplemented
+        return (
+            self.overall_cost == other.overall_cost
+            and self.candidates == other.candidates
+        )
+
+    def is_close(
+        self,
+        other: "BendersSolution",
+        *,
+        rel_tol: float = 1.0e-9,
+        abs_tol: float = 0.0,
+    ) -> bool:
+        return (
+            math.isclose(
+                self.overall_cost, other.overall_cost, abs_tol=abs_tol, rel_tol=rel_tol
+            )
+            and self.candidates.keys() == other.candidates.keys()
+            and all(
+                math.isclose(
+                    self.candidates[key],
+                    other.candidates[key],
+                    rel_tol=rel_tol,
+                    abs_tol=abs_tol,
+                )
+                for key in self.candidates
+            )
+        )
+
+    def __str__(self) -> str:
+        lpad = 30
+        rpad = 12
+
+        string = "Benders' solution:\n"
+        string += f"{'Overall cost':<{lpad}} : {self.overall_cost:>{rpad}}\n"
+        string += f"{'Investment cost':<{lpad}} : {self.investment_cost:>{rpad}}\n"
+        string += f"{'Operational cost':<{lpad}} : {self.operational_cost:>{rpad}}\n"
+        string += "-" * (lpad + rpad + 3) + "\n"
+        for candidate, investment in self.candidates.items():
+            string += f"{candidate:<{lpad}} : {investment:>{rpad}}\n"
+
+        return string
+
+    @property
+    def investment_cost(self) -> float:
+        return self.data["solution"]["investment_cost"]
+
+    @property
+    def operational_cost(self) -> float:
+        return self.data["solution"]["operational_cost"]
+
+    @property
+    def overall_cost(self) -> float:
+        return self.data["solution"]["overall_cost"]
+
+    @property
+    def candidates(self) -> Dict[str, float]:
+        return self.data["solution"]["values"]
+
+    @property
+    def status(self) -> str:
+        return self.data["solution"]["problem_status"]
+
+    @property
+    def optimality_gap(self) -> float:
```

Comment:
Rename function to `absolute_gap` (closer to the mathematical meaning of the data)